app/Console/Services/Importer.php

```diff
@@ -190,8 +199,11 @@ protected function getTotalRows($table)
      */
     protected function readRows($table)
     {
+        // Set the table prefix if it has been entered
+        $this->tableName($table);
+
         $offset = 0;
-        $total_rows = $this->getTotalRows($table);
+        $total_rows = intval($this->getTotalRows($table));
```

Comment:
I wouldn't put this here. In the `getTotalRows()` method, do the `return (int) $rows` I believe it would be